src/main/resources/db/migration/V4_2__standardtekst.sql

```diff
@@ -0,0 +1,14 @@
+-- ROLLBACK-START
+------------------
+-- DROP TABLE STANDARDTEKST;
+---------------
+-- ROLLBACK-END
+
```

Comment:
For å begynne et sted rundt dette med standardtekster så foreslår jeg denne databasetabellen. 

Nokkel-feltet er tenkt å inneholde nøkler fra en Kotlin-enum som må lages i isdialogmote.

I utgangspunktet kan vi legge inn tekst "Ipsum lorum..." med gyldig_fom='1900-01-01' for hver av nøklene med et flyway-script. Og så kan vi lage nye flyway-script etter hvert som vi får tak i de tekstene som skal benyttes (den eksisterende forekomsten får da satt  gyldig_til-feltet, mens den nye raden har samme dato-verdi i gyldig_fom-kolonnen.

Jeg ser for meg at alle endringer kan gjøres med flyway-script, så det vi trenger ellers er et query for å lese ut gjeldende tekst for et nokkel og dato (som kan defaulte til dagens dato), og en get-tjeneste i api'et til isdialogmote for å lese ut tilsvarende.

Reply:
Jeg er ikke sikker på om jeg er fan av å bruke flyway for å skrive data til databasen, det er vel ikke feil, men jeg er mer vant til at flyway er for strukturelle endringer 🤔 Det føles for øvrig litt rart å skulle lagre tekst i database på denne måten, men jeg har ingen alternative løsninger 🤷‍♂️

Reply:
Jeg mener at det er en helt kurant måte å oppdatere tekstene på, og i prosjekter jeg har jobbet på tidligere har vi benyttet flyway på denne måten. Det går fint an å begynne på denne måten, og så evnt innføre post-tjenester for å legge inn nye tekster senere. Men så lenge det er vi i teamet som står for oppdatering av tekstene er det enklere å gjøre det med flyway.

Text-kolonner i Postgres kan ha vilkårlig lange tekster som verdi. Eneste alternativ jeg kan komme på er å ha de som en del av kodebasen, man da tror jeg at jeg foretrekker databasen.

Hvis vi bruker flyway-script vil tekstene være leselige som en del av kodebasen også, men jeg antar at det ikke er noe sensitivt med disse standardtekstene.